### PR TITLE
Custom config for backgroundupdate-allowunelevated-rollout-beta

### PR DESCRIPTION
**Pull request by scholtzan**
https://mozilla-hub.atlassian.net/browse/DENG-1437

Custom OpMon dashboard with crash metrics

### CODE REVIEW

File: opmon/backgroundupdate-allowunelevated-rollout-beta.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[project]
+
+name = "Background Update allowUnelevated Rollout Beta"
+platform = "firefox_desktop"
+xaxis = "submission_date"
+start_date = "2023-09-01"
```
Comment (scholtzan) [inline opmon/backgroundupdate-allowunelevated-rollout-beta.toml:6]: We should update this to when the rollout starts rolling out

File: opmon/backgroundupdate-allowunelevated-rollout-beta.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+metrics = [
+  "content_shutdown_crashes",
+  "content_crashes",
+  "shutdown_hangs",
+  "oom_crashes",
+  "main_crashes",
+  "startup_crashes"
```
Comment (scholtzan) [inline opmon/backgroundupdate-allowunelevated-rollout-beta.toml:13]: This is adding the crash metrics (like the ones seen on the [dashboard linked in the ticket](https://mozilla.cloud.looker.com/dashboards/operational_monitoring::mr2022_backgroundtaskmessage_notification_beta_107?Date=&Percentile=50)). The default guardrail metrics will still show up, but we could remove them if they are not needed.

Comment (scholtzan) [reply opmon/backgroundupdate-allowunelevated-rollout-beta.toml:13]: Also, we can add other metrics that might be of interest. Not sure if something around `background_update.reasons_to_not_update` should be monitored?

**Comment by mikewilli** [comment]
Can this be closed?

**Pull request closed without merging**